### PR TITLE
fix: rollback change to project dist output and remove .png, ref leat…

### PR DESCRIPTION
**Pull request by pete-watters**
This PR contains work to:
- roll back my change to remove `dist-all`
- remove `AvatarIcons` that use `.png`

### `dist-all`
I tried to remove `dist-all` as it seemed un-necessary, however this meant the build misses some `panda.` files and I need to investigate how to solve that. Missing those files means that we miss styles in the extension. I'll investigate that in https://github.com/leather-io/issues/issues/107. 

### Avatar Icons
We have two avatar icons in the library that use `.png`s. When we install the package including those it cannot find them and it corrupts the package. Based on discussions and https://github.com/leather-io/issues/issues/103, I have decided to simply remove these two avatars from the library and keep them in the extenstion pending further investigation 

## Summary by CodeRabbit

- **Documentation**
  - Added setup instructions for integrating Leather Wallet's UI library with panda-css.
  
- **New Features**
  - Introduced new build script `build:all` to streamline asset copying and TypeScript compilation.
  
- **Refactor**
  - Updated export paths in package.json to new `dist-all` directory.
  - Removed export of `StampsAvatarIcon` and `Stx20AvatarIcon` components.

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -25,9 +26,9 @@
   },
   "exports": {
     ".": {
-      "types": "./dist-web/web.d.ts",
-      "import": "./dist-web/web.js",
-      "require": "./dist-web/web.js"
+      "types": "./dist-all/web.d.ts",
```
Comment (pete-watters) [inline packages/ui/package.json:29]: Apologies for the faffing here but when I changed this in https://github.com/leather-io/mono/pull/228 I missed a step / encountered an un-expected issue. 

The `build:all` step was also creating `panda.` files in `dist-all` and I'm still not quite sure how it works so I decided to rollback and finish migrating the library, then optimise it. 

Screenshot 2024-06-20 at 14 20 16

**Review by fbwoolf — APPROVED** [review]

**Review by kyranjamie — COMMENTED** [review]
Sorry I never reviewed, had pending comments. 

LGTM, but README mischaracterises role of UI pkg

File: packages/ui/README.md [hunk 2]
```diff
@@ -1,6 +1,54 @@
...
+    'leather-styles': path.resolve('leather-styles'),
+    'react': path.resolve('./node_modules/react'),
+    'react-dom': path.resolve('./node_modules/react-dom')
+    },
+      ...
+    rules: [
+      ...
+      {
+        test: /\.(js)$/,
+        include: [/node_modules\/@cteic\/ui/],
```
[the remark this answers is not in the document]
Comment (pete-watters) [reply packages/ui/README.md:46]: I'll change this. that's my personal npm org I was using to test the package more quickly. I'll update it

**Comment by pete-watters** [comment]
> Sorry I never reviewed, had pending comments.
> 
> LGTM, but README mischaracterises role of UI pkg

Thanks for the feedback. I have made updates in https://github.com/leather-io/mono/pull/242

**Pull request merged**